### PR TITLE
[feature] Add experience context to orders api call

### PR DESCRIPTION
**Pull request by Reedian**
### WHAT?
Include experiance context in order api call

### WHY?
To serve better experience for buyers through PayPal gateway

Ticket: [PAYM-1033](https://maxioevolution.atlassian.net/browse/PAYM-1033)

[PAYM-1033]: https://maxioevolution.atlassian.net/browse/PAYM-1033?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/active_merchant/billing/gateways/paypal_complete.rb [hunk 1]
```diff
@@ -106,7 +106,18 @@ def add_payment_source(post, vault_id, options)
         post[:payment_source] ||= {}
         payment_source = options[:payment_type] == "paypal_account" ? :paypal : :card
         post[:payment_source][payment_source] = {
-          vault_id: vault_id
+          vault_id: vault_id,
+          experience_context: {
+            payment_method_preference: "IMMEDIATE_PAYMENT_REQUIRED",
+            brand_name: "Maxio",
```
Comment (himn1) [inline lib/active_merchant/billing/gateways/paypal_complete.rb:112]: Is it ok to hard-code this value?

**Review by himn1 — COMMENTED** [review]
left 1

File: lib/active_merchant/billing/gateways/paypal_complete.rb (hunk 1 above)
Comment (Reedian) [reply lib/active_merchant/billing/gateways/paypal_complete.rb:112]: I think it's ok unless we are planning to change company name anytime soon 😅

**Review by himn1 — APPROVED** [review]
🚢

**Pull request merged**